src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -269,7 +270,13 @@ export const Checkout = ({
     createQuoteBundle,
     { loading: isBundleCreationInProgress },
   ] = useCreateQuoteBundleMutation()
-
+  const { loading: isLoadingQuoteCart } = useQuoteCartQuery({
+    variables: {
+      id: quoteCartId,
+      locale: locale.isoLocale,
+    },
+    notifyOnNetworkStatusChange: true,
```

Comment:
Should we make this more explicit by adding something like this also:

```
if (networkStatus === NetworkStatus.refetch) return 'Refetching!'
```

https://www.apollographql.com/docs/react/data/queries/

Reply:
I guess I didn't get it 😬

Reply:
not really, since im not using the networkStatus field, but using notifyOnNetworkStatusChange triggers the correct loading value
@guilhermespopolin without this opt, the query just refreshes the data whenever there's a change, but keeps loading:false